rules_gapic/csharp/csharp_gapic.bzl

```diff
@@ -108,7 +108,7 @@ _csharp_gapic_postprocessed_srcjar = rule(
     },
 )
 
-def csharp_gapic_srcjar(name, src, gapic_yaml, service_yaml, package, **kwargs):
+def csharp_gapic_srcjar(name, src, gapic_yaml, service_yaml, package, grpc_service_config = None, **kwargs):
```

Comment:
Speaking about formatting changes: when the arguments list get that long, we should probably switch to the arg-per-line style. Not very important, but please consider reformatting this to that style.

Reply:
Done!